Dev simple spend

frier17 opened the pull request:
Hi. I am proposing these changes to simple_spend with respect to issue #122  by @haysquareA. I have modified the simple_spend to take a `preference` parameter which can be used as a flag to set the transaction fee as low, medium, zero, or high in creating an unsigned transaction.
I also added update to the simple_spend documentation to reflect the `preference` variable.
Lastly, a test was added as SimpleSpendTX in the `test_blockcypher.py` script. This tests uses example from the blockcypher documentation with values set to fail due to insufficient funds in the address used.

Kindly review.

Thanks.

quentinlesceller commented on blockcypher/api.py:
```diff
@@ -1624,7 +1624,7 @@ def broadcast_signed_transaction(unsigned_tx, signatures, pubkeys, coin_symbol='
 
 def simple_spend(from_privkey, to_address, to_satoshis, change_address=None,
-        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc'):
+        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc', preference: str = 'low'):
```
For a better UX I suggest that keep the fee `high` here.

frier17 replied:
Thanks for your feedback @quentinlesceller. I plan making the needed changes. Also noticed a similar need for the function `simple_spend_p2sh`. Will attempt to make similar changes and add tests as well.

Thanks

quentinlesceller reviewed (changes requested):
Thank you for the PR @frier17. Looking good. One minor comment regarding defaulting to low fees.

quentinlesceller commented:
Oh just noticed the zero fee. Transaction with zero fees will be directly rejected by most transaction pools so I would remove the option.

frier17 commented:
Thanks for the feedback @quentinlesceller. I have removed the `zero` option and updated the tests. Similar work was done on the `simple_spend_p2sh` function.

quentinlesceller reviewed (approved):
Amazing. Thank you for the PR! LGTM!
